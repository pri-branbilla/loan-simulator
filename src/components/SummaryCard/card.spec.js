import React from 'react'
import { SummaryCard } from './index'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <SummaryCard
      installmentAmount="465,00"
      amount={123456}
      feeRate="111,12%"
    />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
