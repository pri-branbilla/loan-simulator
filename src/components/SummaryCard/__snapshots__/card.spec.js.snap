// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="form__result"
>
  <div
    className="quota__container"
  >
    <h4>
      Valor da Parcela
    </h4>
    <div
      className="quota"
    >
      <strong>
        R$ 
      </strong>
      <span>
        465,00
      </span>
    </div>
  </div>
  <div
    className="amount_container"
  >
    <h4>
      Total a pagar
    </h4>
    <p>
      R$123,456.00
    </p>
  </div>
  <div
    className="tax__container"
  >
    <h4>
      Taxa de juros (mÃªs)
    </h4>
    <p>
      111,12%
       %
    </p>
  </div>
  <button
    className="button"
  >
    Solicitar
  </button>
</div>
`;
